docs/scos/user/back-office-user-guides/202108.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -21,38 +21,36 @@ A cart rule is a discount that is applied automatically when all attached discou
 
 ## Prerequisistes
 
-To start working with discounts, go to **Merchandising** > **Discount**.
+To start working with discounts, go to **Merchandising&nbsp;<span aria-label="and then">></span> Discount**.
 
 ## Creating a cart rule discount
 
 To create a cart rule discount:
-1. On the *Discount* page, in the top right corner, click **Create new discount**.
-2. On the *Create Discount* page, in the *General information* tab, do the following:
+1. On the **Discount** page, in the top right corner, click **Create new discount**.
+2. On the **Create Discount** page, on the **General information** tab, do the following:
     1. In **Store relation**, check the stores you wish the discount to be active in.
     2. From the **Discount Type** drop-down list, select a cart rule.
     3. In the **Name** field, specify the name for the discount.
-    4. _Optional_: in the **Description** field, enter the description of the discount.
-    5. Specify if the discount is exclusive. For reference information, in the Creating a voucher guide, see the [Discount Overview page](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-overview-page) section.
+    4. Optional: in the **Description** field, enter the description of the discount.
+    5. Specify if the discount is exclusive. For reference information, in the **Creating vouchers** guide, see the [Discount Overview page](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-overview-page) section.
     6. Specify the validity interval (lifetime) of the discount.
-* Click **Next** or select the *Discount calculation* tab to proceed.
+3. Click **Next** or select the **Discount calculation** tab to proceed.
 
-* On the *Create Discount* page, in the *Discount calculation* tab, do the following:
-    1.  Select either Calculator percentage or Calculator fixed in the **Calculator type** drop-down. For reference information, in the Creating a voucher guide, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+* On the **Create Discount **page, in the **Discount calculation** tab, do the following:
+    1.  Select either Calculator percentage or Calculator fixed in the **Calculator type** drop-down. For reference information, in the **Creating vouchers** guide, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
```

Comment:
We have 3 BO guides in total. One of them contains references, the other two don't, they just have links to the first one. The configuration happens on the same page within the same menus. Therefore, if I add references to the other two guides, it will result in having copy-pasted content across the document.